lock

```diff
@@ -44,7 +47,7 @@ options="Options:
 # move pipefail down as for some reason "convert -list font" returns 1
 set -o pipefail
 trap 'rm -f "$image"' EXIT
-temp="$(getopt -o :hdnpglt:f: -l desktop,help,listfonts,nofork,pixelate,greyscale,text:,font: --name "$0" -- "$@")"
+temp="$(getopt -o :hdnpglt:fb: -l desktop,help,listfonts,nofork,pixelate,greyscale,text:,font,blur: --name "$0" -- "$@")"
```

Comment:
one actually needs `font:` because `font` has a required parameter.

Reply:
Right, fixed it. Thanks for the notice.